tokio-stream/src/stream_ext.rs

```diff
@@ -113,6 +113,12 @@ pub trait StreamExt: Stream {
     /// pinning it to the stack using the `pin_mut!` macro from the `pin_utils`
     /// crate.
     ///
+    /// # Cancel safety
+    ///
+    /// This method is cancel safe. The returned future only
+    /// holds onto a `Pin<&mut Stream>`, so dropping it won't lose
+    /// a value.
```

Comment:
I don't think I would phrase it this way. Talking about pinning seems like it would only confuse the user.